speaking-portal-project/src/animationFactory/ffmpeg.ts

```diff
@@ -5,13 +5,11 @@ import { spawnSync } from 'node:child_process'
 
 export async function ffmpegProcessor(args: string[], fileType: string) {
     const ffmpegProc = spawnSync('ffmpeg', args)
-    // This portion of the message is only included when ffmpeg is successful in generating a video
-    if (!ffmpegProc.stderr.includes(`Output #0, ${fileType}`)) {
+    if (ffmpegProc.status != 0) {
```

Comment:
Should we have more of a log when the exit code !=0?

I was looking at the different things we can get out of it, and although we might not want all of these to console, they're possible things to add!

And do we want to throw the `stderr` contents or the `error` object?

```
    if (rhubarbProc.status != 0){ // exit code of 0 means it worked successfully
        console.log('exit code (status): ', rhubarbProc.status) // exit code of child process
        console.log('stdout is: ', rhubarbProc.stdout) // contents of output[1]
        console.log('stderr is ', rhubarbProc.stderr) // contents of output[2]
        console.log('signal is ', rhubarbProc.signal) // signal used to kill the child process
        throw Error(`${rhubarbProc.error}`) // error object if child process failed or timed out
    }
```

Reply:
I just realized I commented for Rhubarb instead of ffmpeg lol woops

still valid question of if ffmpeg has an error object instead of just stderr (as I'm not as familiar with it)